content/how-to/add-code-servers/bitbucket-cloud/_index.md

```diff
@@ -37,7 +37,7 @@ The next step in adding the code server is adding the Personal Access Token (PAT
 
 Example: `holland:Vgc9Kw8w5zQu58989Uxz`
 
-[How to generate a personal access token on Bitbucket](/how-to/add-code-servers/bitbucket-cloud/generate-pat/)
+[How to generate a personal access token on Bitbucket](/how-to/add-code-servers/bitbucket-cloud/access-token/)
```

Comment:
Change this link, or change the slug in the other file? I thought we agreed `generate-pat` was the consistent choice?

Reply:
Changed the slugs to `generate-pat` and made the links consistent now.